farm/file_utils.py

```diff
@@ -284,13 +284,6 @@ def get_file_extension(path, dot=True, lower=True):
     return ext.lower() if lower else ext
 
 
-# TODO: still needed?
-# Allow retrieval of config args with dot notation
-class Struct:
-    def __init__(self, **entries):
-        self.__dict__.update(entries)
-
-
 def read_config(path, flattend=False):
```

Comment:
Should we move the config related functions to "experiment.py"? or an own config.py?